action.yml

```diff
@@ -15,12 +15,18 @@ env:
 runs:
   using: 'composite'
   steps:
-    - uses: getsentry/action-setup-venv@3a832a9604b3e1a4202ae559248f26867b467cc7 #v2.1.1
+    - uses: astral-sh/setup-uv@884ad927a57e558e7a70b92f2bccf9198a4be546 # v6
+      with:
+        version: '0.8.2'
+        # we just cache the venv-dir directly in action-setup-venv
+        enable-cache: false
```

Comment:
should we include python version here?

Reply:
No, we don't want to use uv-managed pythons in CI, we want to use setup-python's. In some cases, problems happen when you try and use uv-managed pythons in CI.